package-lock.json

```diff
@@ -0,0 +1,16432 @@
+{
+  "name": "async-race",
```

Comment:
name is rslang

Reply:
This name has been changed to another branch